### PR TITLE
Enhance TiDB login descriptions

### PR DESCRIPTION
**Pull request by qiancai**
### What is changed, added or deleted? (Required)

1. Added a sentence to show that the IP address used to log in to TiDB is the tidb_server address configured in the cluster topology file. 
2. Added a sentence to show that the default password of the root user for the TiDB Dashboard login is empty. 

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: post-installation-check.md [hunk 1]
```diff
@@ -48,14 +48,16 @@ tiup cluster display tidb-test
...
+执行以下命令登录数据库：
 
 {{< copyable "shell-regular" >}}
 
 ```shell
-mysql -u root -h 10.0.1.4 -P 4000
+mysql -u root -h ${tidb_server_host_IP_address} -P 4000
 ```
 
+其中 `${tidb_server_host_IP_address}` 是你在[初始化集群拓扑文件](/production-deployment-using-tiup.md#第-3-步初始化集群拓扑文件)时为 `tidb_servers` 配置的 IP 地址，例如 `10.0.1.7`。
```
Comment (qiancai) [inline post-installation-check.md:59]:
```suggestion
其中 `${tidb_server_host_IP_address}` 是在[初始化集群拓扑文件](/production-deployment-using-tiup.md#第-3-步初始化集群拓扑文件)时为 `tidb_servers` 配置的 IP 地址，例如 `10.0.1.7`。

```

File: post-installation-check.md [hunk 2]
```diff
@@ -48,14 +48,17 @@ tiup cluster display tidb-test
...
 
 {{< copyable "shell-regular" >}}
 
 ```shell
-mysql -u root -h 10.0.1.4 -P 4000
+mysql -u root -h ${tidb_server_host_IP_address} -P 4000
 ```
 
+其中 `${tidb_server_host_IP_address}` 是在[初始化集群拓扑文件](/production-deployment-using-tiup.md#第-3-步初始化集群拓扑文件)时为 `tidb_servers` 配置的 IP 地址，例如 `10.0.1.7`。
+
```
Comment (qiancai) [inline post-installation-check.md:60]:
```suggestion
```

File: post-installation-check.md [hunk 3]
```diff
@@ -48,14 +48,16 @@ tiup cluster display tidb-test
...
+执行以下命令登录数据库：
 
 {{< copyable "shell-regular" >}}
 
 ```shell
-mysql -u root -h 10.0.1.4 -P 4000
+mysql -u root -h ${tidb_server_host_IP_address} -P 4000
 ```
 
+其中 `${tidb_server_host_IP_address}` 是在[初始化集群拓扑文件](/production-deployment-using-tiup.md#第-3-步初始化集群拓扑文件)时为 `tidb_servers` 配置的 IP 地址，例如 `10.0.1.7`。
```
Comment (lucklove) [inline post-installation-check.md:59]:
```suggestion
其中 `${tidb_server_host_IP_address}` 是在[初始化集群拓扑文件](/production-deployment-using-tiup.md#第-3-步初始化集群拓扑文件)时为 `tidb_servers` 配置的 IP 地址之一，例如 `10.0.1.7`。

```

**Comment by lucklove** [comment]
REST LGTM

**Review by TomShawn — APPROVED** [review]
LGTM

File: post-installation-check.md [hunk 4]
```diff
@@ -48,14 +48,17 @@ tiup cluster display tidb-test
...
+执行以下命令登录数据库：
 
 {{< copyable "shell-regular" >}}
 
 ```shell
-mysql -u root -h 10.0.1.4 -P 4000
+mysql -u root -h ${tidb_server_host_IP_address} -P 4000
 ```
 
+其中 `${tidb_server_host_IP_address}` 是在[初始化集群拓扑文件](/production-deployment-using-tiup.md#第-3-步初始化集群拓扑文件)时为 `tidb_servers` 配置的 IP 地址之一，例如 `10.0.1.7`。
```
Comment (qiancai) [inline post-installation-check.md:59]:
```suggestion
其中， `${tidb_server_host_IP_address}` 是在[初始化集群拓扑文件](/production-deployment-using-tiup.md#第-3-步初始化集群拓扑文件)时为 `tidb_servers` 配置的 IP 地址之一，例如 `10.0.1.7`。

```

**Comment by qiancai** [comment]
The suggesetion is applied. @lucklove PTAL again, thanks.

**Comment by TomShawn** [comment]
> The suggesetion is applied. @lucklove PTAL again, thanks.

Confirmed with @lucklove, this PR can be merged.

**Pull request merged**